client/src/constants.ts

```diff
@@ -47,6 +47,15 @@ export const MDN_PLUS_SUBSCRIBE_5Y_URL = `${process.env.REACT_APP_MDN_PLUS_SUBSC
 export const MDN_PLUS_SUBSCRIBE_10M_URL = `${process.env.REACT_APP_MDN_PLUS_SUBSCRIBE_URL}?plan=${process.env.REACT_APP_MDN_PLUS_10M_PLAN}`;
 export const MDN_PLUS_SUBSCRIBE_10Y_URL = `${process.env.REACT_APP_MDN_PLUS_SUBSCRIBE_URL}?plan=${process.env.REACT_APP_MDN_PLUS_10Y_PLAN}`;
 
+export const MDN_PLUS_5M_PLAN_NAME =
+  process.env.REACT_APP_MDN_PLUS_5M_PLAN_NAME || "";
+export const MDN_PLUS_5Y_PLAN_NAME =
+  process.env.REACT_APP_MDN_PLUS_5Y_PLAN_NAME || "";
+export const MDN_PLUS_10M_PLAN_NAME =
+  process.env.REACT_APP_MDN_PLUS_10M_PLAN_NAME || "";
+export const MDN_PLUS_10Y_PLAN_NAME =
+  process.env.REACT_APP_MDN_PLUS_10Y_PLAN_NAME || "";
```

Comment:
These constants are already defined in the `SubscriptionType` enum.

Why would we need to make these configurable?

Reply:
Dang, I had no idea about those. Will have a look and change.